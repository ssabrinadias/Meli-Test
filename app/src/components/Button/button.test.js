import React from "react";
import { render } from "@testing-library/react";
import Button from ".";

describe("Button component", () => {
	it("[snapshot] - should be able to render ", () => {
		const { container } = render(<Button />);
		expect(container).toMatchSnapshot();
	});
});
