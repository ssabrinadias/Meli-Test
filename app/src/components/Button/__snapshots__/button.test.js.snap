// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component [snapshot] - should be able to render  1`] = `
<div>
  <button
    class="sc-AxjAm hsMGRn"
  />
</div>
`;
